packages/server/graphql/public/mutations/helpers/movePageToNewParent.ts

```diff
@@ -14,6 +15,10 @@ export const movePageToNewParent = async (
   parentPageId: number
 ) => {
   const pg = getKysely()
+  const viewerAccess = await pageAccessByUserId({} as any).load({pageId, userId: viewerId})
+  if (viewerAccess !== 'owner') {
+    throw new GraphQLError(`Viewer must own the Page in order to move it to a new parent`)
```

Comment:
-1 `GraphQLError`s are potentially user facing, aren't they? So "Viewer" should be "You".